pytest.ini

```diff
@@ -1,4 +1,5 @@
 [pytest]
+testpaths = integration
```

Comment:
This is unrelated, but means `pytest` without arguments doesn't try to run trial tests. I ran into this when testing this branch. In particular, our trial tests have incompatible assumptions about temporary paths, and will write to the current directory when run via pytest.